.github/workflows/ci.yml

```diff
@@ -421,6 +421,36 @@ jobs:
       - name: Test import all
         run: python -c 'from jina import *'
 
+  install-jina-ecosystem-test:
+    runs-on: ubuntu-latest
+    needs: [lint-flake-8, code-injection ]
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2.5.0
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.7
+      - name: Prepare enviroment
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install wheel
+
+          HUBBLE_VERSION=$(curl -L -s "https://pypi.org/pypi/jina-hubble-sdk/json" |  jq  -r '.releases | keys | .[]| select(startswith("0."))'  | sort -V | tail -1)
+          DOCARRAY_VERSION=$(curl -L -s "https://pypi.org/pypi/docarray/json" |  jq  -r '.releases | keys | .[]| select(startswith("0."))'  | sort -V | tail -1)
+          JCLOUD_VERSION=$(curl -L -s "https://pypi.org/pypi/jcloud/json" |  jq  -r '.releases | keys | .[]| select(startswith("0."))'  | sort -V | tail -1)
+
+          pip install . jcloud==$JCLOUD_VERSION docarray==$DOCARRAY_VERSION jina-hubble-sdk==$HUBBLE_VERSION
+        env:
+          JINA_PIP_INSTALL_CORE: ${{ matrix.core }}
+          JINA_PIP_INSTALL_PERF: ${{ matrix.perf }}
+
+      - name: Test basic import
+        run: python -c 'from jina import Executor,requests'
+      - name: Test import all
+        run: python -c 'from jina import *'
+
 
   # just for blocking the merge until all parallel core-test are successful
   success-all-test:
```

Comment:
why not added to all success?

Reply:
It some case we might want to introduce breaking change / causing a dependency conflicts between (Ex annlite and docarray). This test is more to be aware that this we cause a problem rather than blocking you